test/tests/functional/pbs_checkpoint.py

```diff
@@ -144,7 +148,7 @@ def test_checkpoint_abort_with_preempt(self):
         self.server.manager(MGR_CMD_CREATE, QUEUE, a, "expressq")
 
         j1 = Job(TEST_USER, self.attrs)
-        j1.set_sleep_time(20)
+        j1.set_sleep_time(80)
```

Comment:
why did you increase the sleep time of the job?

Reply:
Because we need, both job must be available. but first job get finished before second job is started.

Reply:
Why is it that the first job finishes before second starts? I don't see any place in the test which would cause a delay unless there is some problem with the system itself.
In any case, If you really want to increase the sleep time, I'd say just don't set it. By default, every job sleeps for 100 seconds.

Reply:
Or if you don't need the to wait for the jobs to end, increase it longer than 100 seconds.

Reply:
> Why is it that the first job finishes before second starts? I don't see any place in the test which would cause a delay unless there is some problem with the system itself.
> In any case, If you really want to increase the sleep time, I'd say just don't set it. By default, every job sleeps for 100 seconds.

Some platforms take long time to do job process.

Reply:
Hmmm, that seems odd that PBS could not preempt the job within 20 seconds of running one job when the test does not even have any sleep. In any case, just don't set and it will be set to 100 seconds by default.